glean-core/ios/GleanTests/Scheduler/MetricsPingSchedulerTests.swift

```diff
@@ -2,10 +2,10 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+/*
```

Comment:
And with one fell swoop, @badboy fixes the MPS tests for good!

Reply:
![image](https://img2.thejournal.ie/inline/2372867/original/?width=500&version=2372867)